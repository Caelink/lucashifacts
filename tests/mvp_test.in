#!/bin/bash

export EMAILS=""
export MAIL_SCRIPT="$RUN_DIR/src/disrupt.sh"
export FACTS_SOURCE="$TEST_DIR/mvp_test.lf"
