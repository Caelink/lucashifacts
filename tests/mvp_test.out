<html>
<h1>Hello!</h1>
<p>it is i your dude<br>
thank you for subscribing to<br>
lucashi's lit facts</p>

<p>Did <em>you</em> know that this lucashi fact is a test fact </p>
</html>
